runtime/android/core_internal/src/org/xwalk/core/internal/HitTestResultInternal.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2006 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// Copyright (c) 2016 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package org.xwalk.core.internal;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.util.Map;
+
+@XWalkAPI(createInternally = true)
+public class HitTestResultInternal {
```

Comment:
@fujunwei Should we change this name to XWalkHitTestResultInternal? if so, developper use "XWalkHitTestResult ** = ((XWalkView) v).getHitTestResult();", while webview use case is "
HitTestResult ** = ((XWalkView) v).getHitTestResult();", while webview use case is "
WebView.HitTestResult result = ((WebView) v).getHitTestResult();"